swift_static_framework/swift_static_framework.bzl

```diff
@@ -87,23 +127,58 @@ _swift_static_framework = rule(
             executable = True,
         ),
     ),
+    fragments = ["apple"],
     outputs = {
         "fat_file": "%{name}.fat",
         "output_file": "%{name}.zip",
     },
 )
 
-def swift_static_framework(name, srcs = [], deps = []):
+def _swift_static_framework(name, srcs = [], deps = [], **kwargs):
```

Comment:
did you mean to add the underscore here?

Reply:
Good catch! Fixed.